src/index.js

```diff
@@ -203,13 +221,19 @@ export function validate(config, metaConfig) {
 }
 
 function assertValid(value, validateKey, validator) {
-    if (!assert(value, validator)) {
-        throwError(validateKey);
+    const result = assert(value, validator);
```

Comment:
I see the old use was a bit off as well, in the sense that an assertion itself returns a value of significance. It should not really. If it returns a result of importance it is no longer just an assertion. I think "isValid" is a better name here, if we want to evaluate the result of the function. It's not a big deal, it can just be offputting to see an assertion assigned to a variable as one would usually just expect them to throw or not.